sdk/core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -53,62 +58,110 @@ object Klaviyo {
      * (e.g. after a fresh login)
      *
      * @param phoneNumber Phone number for active user
+     * @return
      */
     fun setPhoneNumber(phoneNumber: String): Klaviyo =
-        this.setProfile(Profile().setPhoneNumber(phoneNumber))
+        this.setProfileAttribute(ProfileKey.PHONE_NUMBER, phoneNumber)
+
+    /**
+     * @return The phone number of the currently tracked profile, if set
+     */
+    fun getPhoneNumber(): String? = UserInfo.phoneNumber.ifEmpty { null }
```

Comment:
I wonder if we should ignore empty?

Reply:
I went back and forth, but decided that null vs empty string was potentially useful distinction for the app developer.